cli/install_ee/install_ee_test.go

```diff
@@ -30,9 +31,34 @@ func TestGetVersions(t *testing.T) {
 			err:    fmt.Errorf("no packages for this OS"),
 		}
 
-	inputData1 := []byte(`<a href="/enterprise/tarantool-enterprise-bundle-` +
-		`1.10.10-52-g0df29b137-r419.tar.gz">tarantool-enterprise-bundle-1.10.10-` +
-		`52-g0df29b137-r419.tar.gz</a>                                      2021-08-18 ` +
+	arch, err := util.GetArch()
+	assert.Nil(err)
```

Comment:
Suppose, it must be `require`. And there is a special function for errors: NoError. It is more readable. Don't use Nil to check if no error.

Reply:
Changed

Reply:
Please use require instead of assert here. Suppose, the test should not continue if GetArch or GetOs fails.